src/api/exchange/ExchangeApi.ts

```diff
@@ -109,6 +121,38 @@ export class ExchangeApiImpl extends DepositApiImpl implements ExchangeApi {
     return decodeAuctionElements(encodedOrders)
   }
 
+  public async getOrdersPaginated({
+    userAddress,
+    networkId,
+    offset,
+    pageSize,
+  }: GetOrdersPaginatedParams): Promise<GetOrdersPaginatedResult> {
+    const contract = await this._getContract(networkId)
+    const _pageSize = pageSize || DEFAULT_ORDERS_PAGE_SIZE
+
+    log(
+      `[ExchangeApiImpl] Getting Orders Paginated for account ${userAddress} with offset ${offset} and pageSize ${_pageSize}`,
+    )
+
+    const encodedOrders = await contract.methods.getEncodedUserOrdersPaginated(userAddress, offset, _pageSize).call()
+
+    // is null if Contract returns empty bytes
+    if (!encodedOrders) return { orders: [] }
```

Comment:
we never ended up adding the curly bracket rule?

Reply:
Nope. Still in the backlog https://github.com/gnosis/dex-react/issues/159